src/kotlin_instance.cpp

```diff
@@ -47,7 +48,7 @@ void KotlinInstance::get_method_list(List<MethodInfo>* p_list) const {
 }
 
 bool KotlinInstance::has_method(const StringName& p_method) const {
-    return false;
+    return GDKotlin::get_instance().find_class_by_class_name(wrappedObject->get_class_name())->get_method(p_method) != nullptr;
```

Comment:
maybe `wrappedObject->get_class()` is better? It can call `find_class_by_class_name` under the hood.

Reply:
I can't, this method would return a KtClass, and kt_class import kt_object, so this would introduce a cyclic dependency.